zuul.d/projects.yaml

```diff
@@ -90,25 +86,21 @@
     name: cloudfoundry/bosh-openstack-cpi-release
     periodic-4/16:
       jobs:
-        - bosh-openstack-cpi-release-acceptance-test:
+        - bosh-openstack-cpi-release-v42-acceptance-test-stein:
             branches: master
-        - bosh-openstack-cpi-release-acceptance-test-stein:
+        - bosh-openstack-cpi-release-v42-acceptance-test-rocky:
             branches: master
-        - bosh-openstack-cpi-release-acceptance-test-rocky:
+        - bosh-openstack-cpi-release-v42-acceptance-test-queens:
             branches: master
-        - bosh-openstack-cpi-release-acceptance-test-queens:
+        - bosh-openstack-cpi-release-v42-acceptance-test-pike:
             branches: master
-        - bosh-openstack-cpi-release-acceptance-test-pike:
-            branches: master
-        - bosh-openstack-cpi-release-acceptance-test-ocata:
+        - bosh-openstack-cpi-release-v42-acceptance-test-ocata:
             branches: master
 
 - project:
     name: kubernetes/cloud-provider-openstack
     periodic-4/16:
       jobs:
-        - cloud-provider-openstack-acceptance-test-e2e-conformance:
```

Comment:
Please do not remove the job, cpo project need the periodic job

Reply:
ok, thanks.